src/components/Envelope.vue

```diff
@@ -144,6 +156,9 @@ export default {
 		Moment,
 		MoveModal,
 	},
+	directives: {
+		draggableEnvelope: DraggableEnvelopeDirective,
```

Comment:
…

```suggestion
		draggableEnvelope,
```

for concise code :)